src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentContextFactory.cs

```diff
@@ -10,5 +10,16 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer
     internal abstract class DocumentContextFactory
     {
         public abstract Task<DocumentContext?> TryCreateAsync(Uri documentUri, CancellationToken cancellationToken);
+
+        public async Task<DocumentContext> GetRequiredAsync(Uri documentUri, CancellationToken cancellationToken)
```

Comment:
This name would make sense in Roslyn, but I think here this should probably just be `CreateAsync` to match `TryCreateAsync`